Extension/src/LanguageServer/extension.ts

```diff
@@ -184,7 +184,7 @@ export async function activate(): Promise<void> {
 
     disposables.push(vscode.workspace.onDidChangeConfiguration(onDidChangeSettings));
     disposables.push(vscode.window.onDidChangeActiveTextEditor(onDidChangeActiveTextEditor));
-    ui.activeDocumentChanged(); // Handle already active documents (for non-cpp files that we don't register didOpen).
+    void ui.activeDocumentChanged(); // Handle already active documents (for non-cpp files that we don't register didOpen).
```

Comment:
Is this void needed?

Reply:
...or why was it added?

Reply:
It doesn't appear needed to me since ui.activeDocumentChanged() is a void function already.